src/NUnitFramework/testdata/SetUpFixtureData.cs

```diff
@@ -610,3 +610,154 @@ public void Test()
     }
 }
 #endregion
+
+#region SomeFixtureWithExceptions
+[TestFixture]
+public class SomeBaseFixtureExceptionInSetUp
+{
+	private readonly bool exceptionInFixtureSetUp;
+	private readonly bool exceptionInSetUp;
+	public SomeBaseFixtureExceptionInSetUp(): this(false, true)
+	{		
+	}
+	public SomeBaseFixtureExceptionInSetUp(bool exceptionInFixtureSetUp, bool exceptionInSetUp)
+	{
+		this.exceptionInFixtureSetUp = exceptionInFixtureSetUp;
+		this.exceptionInSetUp = exceptionInSetUp;
+	}
+	[TestFixtureSetUp]
```

Comment:
`TestFixtureSetUp` is obsolete. You should use `OneTimeSetUp` instead. Same with `TestFixtureTearDown` below. This will introduce warnings into the build.